### PR TITLE
[feat]: added country flag - Net PopUp Module

### PR DESCRIPTION
**Pull request by altrdev**
#444 Country flag icon near public ip

### CODE REVIEW

File: Modules/Net/popup.swift [hunk 1]
```diff
@@ -260,6 +260,15 @@ internal class Popup: NSStackView, Popup_p {
         })
     }
     
+    private func flag(country:String) -> String {
+        let base : UInt32 = 127397
+        var s = ""
+        for v in country.unicodeScalars {
+            s.unicodeScalars.append(UnicodeScalar(base + v.value)!)
```
Comment (exelban) [inline Modules/Net/popup.swift:267]: Hi. Are you sure that `UnicodeScalar` cannot be nil?

**Comment by exelban** [comment]
Hi. I'm going to close the issue at Sanday if there will be no changes.

File: Modules/Net/popup.swift (hunk 1 above)
Comment (altrdev) [reply Modules/Net/popup.swift:267]: Cannot be nil. But can be no flag symbol. Closed

Comment (exelban) [reply Modules/Net/popup.swift:267]: [UnicodeScalar](https://developer.apple.com/documentation/swift/unicode/scalar/2905753-init):
`If v is an invalid Unicode scalar value, the result is nil.`

**Pull request closed without merging**